docs/configuring-playbook-bridge-mautrix-signal.md

```diff
@@ -93,7 +93,15 @@ As seen in the mentioned [upstream-documentation](https://docs.mau.fi/bridges/py
 - `allow: true` the bridge won't enable encryption on its own, but will work in encrypted rooms
 - `default: true` the bridge will automatically enable encryption in new portals.
 
-**Note**: [Upstream-documentation](https://docs.mau.fi/bridges/python/signal/index.html) mentions to make sure using postgres if enabling the bridge in encrypted rooms.
+**Note**: 
+* [Upstream-documentation](https://docs.mau.fi/bridges/python/signal/index.html) mentions to make sure using postgres if enabling the bridge in encrypted rooms.
+* Careful when setting `matrix_mautrix_signal_configuration_extension_yaml:`: If you already used this item before for setting permissions add the part: 
+```
+    encryption:
+	  allow: true
+      default: true
+```
+below the permission-part.
```

Comment:
Indentantion is broken in the YAML block (mixing tabs and spaces).

You also seem to be basing this PR on some older commit.. We don't have an **Enable End-to-End-Encryption** section in `docs/configuring-playbook-bridge-mautrix-signal.md` anymore.

We do have a a new docs page that applies to all mautrix bridges: `docs/configuring-playbook-mautrix-bridges.md`, which explains how to enable end-to-end encryption.

It'd be better if we link to `docs/configuring-playbook-mautrix-bridges.md` from each mautrix bridge page, to make it more discoverable. It seems like we're not doing it yet.